### PR TITLE
Set up a 'Hello world' project with two apps

### PR DESCRIPTION
**Pull request by K0ppai**
Hello reviewers,
I've worked on the following in this pull request,
 - Create a new Rails API app called 'hello-rails-back-end'.
 - Initialize project with Git.
 - Make sure that project has a Postgres database set up. 
 - Create a table for storing messages and create 5 different greetings. 
 - Create an API endpoint that selects a random greeting from table.

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @K0ppai , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that the frontend repo does not have professional readme file. Kindly make the readme professional and ask for review again

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by K0ppai** [comment]
Hello @Qoosim , kindly check feature branches for readme files. I've already updated them in case you didn't see it.

File: app/controllers/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class GreetingsController < ApplicationController
+  before_action :set_greeting, only: %i[show update destroy]
+
+  # GET /greetings
+  def index
+    @greetings = Greeting.all
+
+    render json: @greetings
+  end
```
Comment (ghost) [inline app/controllers/greetings_controller.rb]: - [ ] 👍 Great job on incorporating the random greeting feature on the front end! 😊 However, please keep in mind that the API endpoint is expected to provide a random greeting. You can achieve this by defining the index action in your GreetingsController to return a random greeting, just like this:
```
class GreetingsController < ApplicationController
  def index
    @random_greeting = Greeting.order("RANDOM()").first
    render json: { greeting: @random_greeting&.content }
  end
end

```

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @K0ppai , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Route is created
✅ Greetings model is created 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @K0ppai 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Both apps connected properly 👍 
* Api works well 👍 
* Well-written PR 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**